subsys/bluetooth/Kconfig

```diff
@@ -155,6 +155,18 @@ source "subsys/bluetooth/shell/Kconfig"
 
 endif # BT_HCI
 
+config BT_COMPANY_ID
+	hex "Company Id"
+	default 0x05F1
+	range 0x0000 0xFFFF
+	help
+	  Set the Bluetooth Company Identifier for this device. The Linux
```

Comment:
Please add a reference to `BT_CTLR_COMPANY_ID` here so that users know that it exists

Reply:
Updated.